README.md

```diff
@@ -14,13 +14,25 @@ Please read our [Contributing Guidelines](.github/CONTRIBUTING.md), [Code of Con
 ## Setup
 To start contributing to the project, setup the backend environment on your local machine by following the instructions on the [BIT Development Environment Setup Instruction]( https://github.com/anitab-org/bridge-in-tech-backend/wiki/BIT-development-environment-setup) wiki page.
 
+If you prefer to test the latest approved and merged version of the project on the remote server, you can use the [BridgeInTech Heroku Swagger UI server](https://bridgeintech-bit-heroku-psql.herokuapp.com) that already connected to the modified version of Mentorship System heroku backend server made specifically for BridgeInTech development.
 
 ## Branches
 
 This repository has the following branches:
 - **master**: This branch contains the deployment of the backend.
 - **develop**: This contains the latest code. All the contributing PRs must be sent to this branch.
 
+## Project Documentations
+
+This project has live documents that contain information on:
+- Planning - [BridgeInTech Proposal Review](https://docs.google.com/document/d/1uCDCWs8Xyo-3EaUnrLOs48mefXJIN235b1aAswA-s-Y/edit) 
+- [Technical Discussion](https://docs.google.com/document/d/1Fi_dvc1f-J0uuTzzzU7pwZV8hAI9iTX5LlI_THcg_ks/edit#heading=h.gfxf99xhgujm) and Decision made throughout the development
+- [Weekly Scrum progress updates](https://docs.google.com/document/d/1GFeYF7LQV7gG0Cu0lE6Yuit0FVhuUxV-B6c1jBxbro8/edit#heading=h.slbnz8ddhmhh)
```

Comment:
Is this Weekly Scrum update is needed 🤔  I am not sure? That can be accessible via Wiki, right? I would keep the information strictly related to the project in the README, and as you already have the wiki link here, it will be accessible :)

Reply:
Ok, @isabelcosta . I'll remove it then. What about the Meeting Minutes? Should we keep it on README or remove (coz it's also on wiki)?

Reply:
This one has real discussions about the project and not just about the tasks you as a student do throughout the week. Makes sense ?

Reply:
ok, gotcha. I'll keep it there then.